### PR TITLE
(imp) [SD-3087] - Add a Done button on each tab in the Desk editing modal

### PR DESCRIPTION
**Pull request by sjunaid**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

File: client/app/scripts/superdesk-desks/desks.js [hunk 1]
```diff
@@ -868,7 +867,7 @@
                         scope.desk.edit = _.create(desk);
                     };
 
-                    scope.save = function(desk) {
+                    scope.save = function(desk, done) {
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/desks.js]: Docstring would be useful, explaining (among other things), for example,  what the `done` parameter represents.

**Comment by plamut** [comment]
Looks fine, I would just consider adding a few docstrings to the modified methods.

**Comment by sivakuna-aap** [comment]
@sjunaid if you are planning to add docstring, then can you please take care of this as well? https://github.com/superdesk/superdesk/pull/1321#discussion_r44380150

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**